benchmarks/nemo/nemolite2d/manual_versions/psykal_kokkos/Makefile

```diff
@@ -10,23 +10,56 @@
 # export OMPFLAGS=" -fopenmp"
 # export AR=ar
 
-# Additionally it needs the path to a Kokkos installation and execution
-# devices
-KOKKOS_PATH ?= ${HOME}/build/kokkos
-KOKKOS_DEVICES=OpenMP  # Currently this can only be OpenMP
+# Additionally it needs the path to the Kokkos source code
+KOKKOS_PATH ?= $(HOME)/tools/kokkos
+KOKKOS_DEBUG ?= yes
+
+# If no KOKKOS_DEVICES is specified, by default use the OpenMP
+ifndef $(KOKKOS_DEVICES)
+KOKKOS_DEVICES=OpenMP
```

Comment:
Can you just do `KOKKOS_DEVICES? = OpenMP` here?

Reply:
Done